Gopkg.toml

```diff
@@ -75,3 +75,7 @@
 [[override]]
   name = "k8s.io/client-go"
   revision = "1a26190bd76a9017e289958b9fba936430aa3704"
+
+[[constraint]]
+  name = "github.com/coreos/go-systemd"
+  version = "19.0.0"
```

Comment:
Is this an actual constraint? or can we remove this and just handle this dep in Gopkg.lock?

Reply:
Nah, any version should be ok; removed.